frame/benchmarking/src/tests.rs

```diff
@@ -125,6 +125,8 @@ fn new_test_ext() -> sp_io::TestExternalities {
 	GenesisConfig::default().build_storage().unwrap().into()
 }
 
+// NOTE: This attribute is only needed for the `modify_in_` functions.
+#[allow(unreachable_code)]
```

Comment:
Is this because of code that is being generated by the macro?

Reply:
Yep. It is placing code after the `return Err(…)`.